tests/search/components/TestSearchResults.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import SearchResults from 'search/components/SearchResults';
+
+const render = ReactTestUtils.renderIntoDocument;
```

Comment:
I was playing with some stuff locally and I'm thinking we should just pull this out at the require level.

```js
import { renderIntoDocument, isDOMComponent } from 'react-addons-test-utils';